Add Philio PAN07 device

andrey-git opened the pull request:
Add Philio PAN07 device.

This is a unique device which seems not to be fully approved by Zwave alliance, but there are developer sample around on the market.

nechry commented on config/philio/pan07.xml:
```diff
@@ -0,0 +1,57 @@
+			<Help>If the setting is configured for 1hour (set value =6), the device will report its power consumption in KWh every 1 hour to Z-Wave Controller. Unit: 10 minutes.</Help>
+		</Value>
+
+		<Value type="byte" instance="1" index="3" genre="config" label="Selected end point" min="0" max="3" value="1">
```
use a list type

nechry commented on config/philio/pan07.xml:
```diff
@@ -0,0 +1,57 @@
+			 <Help>1: Relay 1 and Relay 2. 2: Relay 1. 3: Relay 2</Help>
+		</Value>
+	
+		<Value type="byte" instance="1" index="4" genre="config" label="Edge or Pulse mode or Edge-Togle mode" min="0" max="3" value="1">
```
it a list but you use a byte type. in list you have to specify the size

nechry commented on config/philio/pan07.xml:
```diff
@@ -0,0 +1,57 @@
+			 <Help>Threshold of kWh for Load Caution. Unit 1KWh</Help>
+		</Value>
+
+		<Value type="byte" instance="1" index="7" genre="config" label="Relay control mode" min="1" max="2" value="1">
```
it a list but you use a byte type. in list you have to specify the size

nechry commented on config/philio/pan07.xml:
```diff
@@ -0,0 +1,57 @@
+
+		<Value type="byte" instance="1" index="7" genre="config" label="Relay control mode" min="1" max="2" value="1">
+		        <Help>Relay control mode. Default: Power mode</Help>
+		        <Item label=" Relay controlling is referred to the meter wattage: When PAN07/PAN09
```
item label must be short, few words of a short sentence

nechry commented on config/philio/pan07.xml:
```diff
@@ -0,0 +1,57 @@
+			<Item label="Control relay directly: Whenever PAN07/PAN09 get an SWITCH ON
+				instruction, it will close the relay; on the other hand, it will open the relay" value="2" />
+		</Value>
+		<Value type="byte" instance="1" index="8" genre="config" label="Restore relay state mode" min="0" max="2" value="1">
```
it's a list but you use a byte type. in list you have to specify the size

nechry commented on config/philio/pan07.xml:
```diff
@@ -0,0 +1,57 @@
+                        <Item label="Last relay switch state" value="1" />
+                        <Item label="NC terminal" value="2" />
+                </Value>
+		<Value type="byte" instance="1" index="9" genre="config" label="Existence of endpoint 3" min="1" max="2" value="1">
```
use a list type instead

nechry commented on config/philio/pan07.xml:
```diff
@@ -0,0 +1,57 @@
+			<Help>1: Exists, 2: Doesn't exists.</Help>
+		</Value>
+	</CommandClass>
+</Product>
```
any associations groups ?

andrey-git replied:
Probably there are associations, but the document doesn't describe them directly.

andrey-git commented:
Addressed all comments

nechry commented:
Your linked file, http://telldus.se/wp-content/uploads/311664-PAN07-1-2-3A-manual-20141028.pdf explains the 3 groups please add it

andrey-git commented:
OZW detects that there are 3 groups even without the config and the config doesn't say what the groups are.

nechry commented:
Exactly and is why you must describe. Grp 1,2 and3 is useless if we don't know the features

andrey-git commented:
Added groups.

Also added
```<CommandClass id="96" mapping="endpoints" />```
Which makes this device look better, although it still makes all reports on the main instance and not on endpoint.
